recipes/example/meta-compiled.yaml

```diff
@@ -0,0 +1,103 @@
+# This recipe example is for packages which are compiled libraries and binaries.
+
+# Jinja variables help maintain the recipe as you'll update the version only
+# here. Using the name variable with the URL in line 7 is convenient when
+# copying and pasting from another recipe, but not really needed.
+{% set name = "simplejson" %}
+{% set version = "3.8.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The URL for GitHub releases
+  url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  sha256: 2b3a0c466fb4a1014ea131c2b8ea7c519f9278eba73d6fcb361b7bdb4fd494e9
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Use separate bld.bat and build.sh files to define the build steps. By
```

Comment:
I'd also clarify this a little further to say it is possible to list the build steps here instead of separate file (especially if it is just `cmake ..`)

Reply:
Either way, I would also include a supplementary build.sh file anyway